src/models/user.model.ts

```diff
@@ -19,8 +19,4 @@ export const UserModel = {
     type: DataTypes.INTEGER,
     allowNull: false,
   },
-  isDeleted: {
```

Comment:
why you removed that?

Reply:
sequelize has it's own soft-delete mechanism with "deletedAt" field